templates/shared_services/sonatype-nexus-vm/terraform/locals.tf

```diff
@@ -1,8 +1,10 @@
 locals {
   core_vnet                = "vnet-${var.tre_id}"
   core_resource_group_name = "rg-${var.tre_id}"
-  nexus_allowed_fqdns      = "*pypi.org,files.pythonhosted.org,security.ubuntu.com,archive.ubuntu.com,keyserver.ubuntu.com,repo.anaconda.com,*.docker.com,*.docker.io,conda.anaconda.org,azure.archive.ubuntu.com,packages.microsoft.com,repo.almalinux.org,download-ib01.fedoraproject.org"
+  nexus_allowed_fqdns      = "*pypi.org,files.pythonhosted.org,security.ubuntu.com,archive.ubuntu.com,keyserver.ubuntu.com,repo.anaconda.com,*.docker.com,*.docker.io,conda.anaconda.org,azure.archive.ubuntu.com,packages.microsoft.com,repo.almalinux.org,download-ib01.fedoraproject.org,cran.r-project.org,cloud.r-project.org"
   nexus_allowed_fqdns_list = distinct(compact(split(",", replace(local.nexus_allowed_fqdns, " ", ""))))
+  windows_nexus_allowed_fqdns = "r3.o.lencr.org,x1.c.lencr.org,*.digicert.com,ocsp.godaddy.com,crl.godaddy.com"
```

Comment:
This is a new set of FQDNs that are allowed from the user-resource VMs?
If so, maybe a name like `workspace_vm_allowed_fqdns` would be more descriptive?

Reply:
That would be more correct, changed to workspace_vm_allowed_fqdns.